src/test/groovy/com/palantir/gradle/versions/GradleTestVersions.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.versions;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+public final class GradleTestVersions {
+    private GradleTestVersions() {}
+
+    public static final List<String> GRADLE_VERSIONS = ImmutableList.of("5.6.3", "6.0.1");
```

Comment:
changelog claims we support 5.3 but we don't test against it. wanna either dial this down or changelog up?